router/router.go

```diff
@@ -173,6 +173,25 @@ func New(cfg *config.Configuration) (r *Router, err error) {
 	disabledBidders := map[string]string{
 		"indexExchange": "Bidder \"indexExchange\" has been deprecated and is no longer available. Please use bidder \"ix\" and note that the bidder params have changed.",
 	}
+	bidderList := openrtb_ext.BidderList()
```

Comment:
I don't see any tests for this... it's enough code that there should probably be some.

I'd recommend splitting it into its own function anyway... both to help make it testable and to avoid adding lines to this already-large function. Could do something like:

`disabledBidders := readDisabledBidders(cfg.Adapters)`


On an unrelated note: code is more readable if you declare variables as close to the point where they're used as possible. You're not really using this until `adapters.ParseBidderInfos(p, bidderList)`, so... put it down there.

Reply:
I have refactored this code.